src/AspNet.Security.OAuth.Kloudless/KloudlessAuthenticationOptions.cs

```diff
@@ -0,0 +1,48 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using static AspNet.Security.OAuth.Kloudless.KloudlessAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.Kloudless
+{
+    /// <summary>
+    /// Defines a set of options used by <see cref="KloudlessAuthenticationHandler"/>.
+    /// </summary>
+    public class KloudlessAuthenticationOptions : OAuthOptions
+    {
+        public KloudlessAuthenticationOptions()
+        {
+            ClaimsIssuer = KloudlessAuthenticationDefaults.Issuer;
+            CallbackPath = KloudlessAuthenticationDefaults.CallbackPath;
+
+            AuthorizationEndpoint = KloudlessAuthenticationDefaults.AuthorizationEndpoint;
+            TokenEndpoint = KloudlessAuthenticationDefaults.TokenEndpoint;
+            UserInformationEndpoint = KloudlessAuthenticationDefaults.UserInformationEndpoint;
+
+            ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, "id");
```

Comment:
Just checking, there's no email-related claim here - is that correct, or an oversight?

Reply:
As value are provided by multi different provider, email address is not guaranteed.

Reply:
But if it is provided it won't be mapped. If there's a consistent claim name for it if it is present, then it would be automatically present rather than needing the user to do it explicitly?

Reply:
Sorry I was not clear enough. Field "account" may contain an email or a different type of "name" depending of the part and/or provider that you use on kloudless.
So I mapped it as "Name".